sentry.edge.config.ts

```diff
@@ -5,8 +5,12 @@
 
 import * as Sentry from "@sentry/nextjs";
 
+import { isProd } from "./lib/utils";
+
 Sentry.init({
   dsn: "https://3a0e99805b57493897f89cae698fe7e9@o4503920068263936.ingest.sentry.io/4505249228324864",
+  environment: process.env.NEXT_PUBLIC_VERCEL_ENV,
```

Comment:
Note that it's not set in env variables. 
And as we disabled it on non prod envs, env is always prod :p